src/Process/Runner/InitialTestsRunner.php

```diff
@@ -66,16 +71,35 @@ public function run(
         array $phpExtraOptions,
         bool $skipCoverage
     ): Process {
+        /** @var Process $process */
         $process = $this->processBuilder->createProcess(
             $testFrameworkExtraOptions,
             $phpExtraOptions,
             $skipCoverage
         );
 
+        //Tracking Process Error Exit
+        $expirationData = (object) [
```

Comment:
is there any reason why to use an object here? It feels like this could just be `$expirationTime = null` and then pass the variable by reference in the callback

Reply:
Bad past experiences is the only reason I do this. I've had too many issues with forced by-ref causing issues that I'd rather rely on the natural by-ref of objects. 

Updated